pkg/api/types/image_types.go

```diff
@@ -0,0 +1,137 @@
+	EstargzMinChunkSize int
+	// EstargzExternalToc separate TOC JSON into another image (called "TOC image"). The name of TOC image is the original + "-esgztoc" suffix. Both eStargz and the TOC image should be pushed to the same registry. (requires stargz-snapshotter >= v0.13.0) (EXPERIMENTAL)
+	EstargzExternalToc bool
+	// EstargzKeepDiffID convert to esgz without changing diffID (cannot be used in conjunction with '--estargz-record-in'. must be specified with '--estargz-external-toc')
+	EstargzKeepDiffID bool
+	// #endregion
+
+	// #region zstd:chunked flags
+	// ZstdChunked convert legacy tar(.gz) layers to zstd:chunked for lazy pulling. Should be used in conjunction with '--oci'
+	ZstdChunked bool
+	// ZstdChunkedCompressionLevel zstd compression level
+	ZstdChunkedCompressionLevel int
+	// ZstdChunkedChunkSize zstd chunk size
+	ZstdChunkedChunkSize int
+	// ZstdChunkedRecordIn read 'ctr-remote optimize --record-out=<FILE>' record file (EXPERIMENTAL)
+	ZstdChunkedRecordIn string
+	// #endregion
+
+	// #region nydus flags
+	// Nydus convert legacy tar(.gz) layers to nydus for lazy pulling. Should be used in conjunction with '--oci'
+	Nydus bool
+	// NydusBuilderPath the nydus-image binary path, if unset, search in PATH environment
+	NydusBuilderPath string
+	// NydusWorkDir work directory path for image conversion, default is the nerdctl data root directory
+	NydusWorkDir string
+	// NydusPrefetchPatterns the file path pattern list want to prefetch
+	NydusPrefetchPatterns string
+	// NydusCompressor nydus blob compression algorithm, possible values: `none`, `lz4_block`, `zstd`, default is `lz4_block`
+	NydusCompressor string
+	// #endregion
+
+	// #region overlaybd flags
+	// Overlaybd convert tar.gz layers to overlaybd layers
+	Overlaybd bool
+	// OverlayFsType filesystem type for overlaybd
+	OverlayFsType string
+	// OverlaydbDBStr database config string for overlaybd
+	OverlaydbDBStr string
+	// #endregion
+
+	// #region generic flags
+	// Uncompress convert tar.gz layers to uncompressed tar layers
+	Uncompress bool
+	// Oci convert Docker media types to OCI media types
+	Oci bool
+	// #endregion
+
+	// #region platform flags
+	// Platforms convert content for a specific platform
+	Platforms []string
+	// AllPlatforms convert content for all platforms
+	AllPlatforms bool
+	// #endregion
+}
+
+type ImageCryptCommandOptions struct {
+	GOptions GlobalCommandOptions
+	// Platforms convert content for a specific platform
+	Platforms []string
+	// AllPlatforms convert content for all platforms
+	AllPlatforms bool
+	// GpgHomeDir the GPG homedir to use; by default gpg uses ~/.gnupg"
+	GpgHomeDir string
+	// GpgVersion the GPG version ("v1" or "v2"), default will make an educated guess
+	GpgVersion string
+	// Keys a secret key's filename and an optional password separated by colon; this option may be provided multiple times"\
```

Comment:
remove `this option may be provided multiple times"\` (helpful as command usage but here we're using a `[]string`, so not need)